### PR TITLE
added "Get Qute" web banner

### PR DESCRIPTION
**Pull request by QMaury**

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -29,6 +29,8 @@ information. Depending on your sign-up date and how long you keep a certain
 level, you can get qutebrowser t-shirts, stickers and more!
 // QUTE_WEB_HIDE_END
 
+image:https://qmaury.com/images/banners/microbanners/quteget.gif["qutegetBanner",link="https://qmaury.com/images/banners/microbanners/quteget.gif"]
+
```
Comment (mschilli87) [inline README.asciidoc:33]: Maybe it is too early and I am totally misreading this, but wouldn't this result in an image linking to itself? I'd expect to be taken to some sort of download page if I clicked on a banner like this.

Comment (QMaury) [reply README.asciidoc:33]: yes, maybe you have a point.
I didn't really think too much about where it really linked, I guess my main idea was just to have it there.
Here, lemme link that to the Qute download page :)

**Review by QMaury — COMMENTED** [review]
banner image now leads to Qutebrowser's install page

**Comment by toofar** [comment]
I'm going to go ahead and say thanks, but no thanks at this point. For the following reasons:

1. hotlinking to an external site is not a good idea for both security and continuity reasons
2. hotlinking to your personal site is out of the question for the same reasons 
3. there are no words on the PR or commit message describing the motivation for this change (and no linked issue) but I don't believe potential users should have any issues finding the install docs as it currently stands because they are linked in ALL CAPS in the next section

A couple of nitpicks about the image anyway:
1. I'm not really into design but despite the project having a globe in the logo I don't think the button fits well with the visual style of the project
2. It's called qutebrowser, lower case and all spelled out fully

**Pull request closed without merging**